ionoscloud/resource_server.go

```diff
@@ -1092,6 +1110,33 @@ func resourceServerDelete(ctx context.Context, d *schema.ResourceData, meta inte
 	return nil
 }
 
+func deleteAllVolumes(ctx context.Context, d *schema.ResourceData, meta interface{}, client *ionoscloud.APIClient, server *ionoscloud.Server) diag.Diagnostics {
```

Comment:
@cristiGuranIonos make sure to get the latest changes from master, this was modified.

Reply:
good point, nothing changed here except position

Reply:
@cristiGuranIonos not really, the name for this function was changed and also the content a little bit.